platform/android/export/export.cpp

```diff
@@ -2334,7 +2334,7 @@ class EditorExportPlatformAndroid : public EditorExportPlatform {
 			_write_tmp_manifest(p_preset, p_give_internet, p_debug);
 
 			//stores all the project files inside the Gradle project directory. Also includes all ABIs
-          _clear_assets_directory();
+			_clear_assets_directory();
```

Comment:
This should ideally be amended in the first commit to avoid unnecessary diffs

Reply:
Done!